clar2wasm/src/wasm_generator.rs

```diff
@@ -1250,8 +1250,23 @@ impl WasmGenerator {
                 .get_expr_type(expr)
                 .ok_or_else(|| GeneratorError::TypeError("constant must be typed".to_owned()))?
                 .clone();
+            let value_length = get_type_size(&ty);
+
+            let (name_offset, name_length) = self.add_string_literal(name)?;
+
+            // Push constant attributes to the stack.
+            builder
+                .i32_const(name_offset as i32)
+                .i32_const(name_length as i32)
+                .local_get(offset_local)
+                .i32_const(value_length);
+
+            // Call a host interface function to load
+            // constant attributes from a data structure.
+            builder.call(self.func_by_name("stdlib.load_constant"));
```

Comment:
Can you open an optimization issue to add the caching that we discussed? Where instead of calling the host interface function every time, it would only call it once, and set a byte in the Wasm memory to indicate that the constant has been loaded.

Reply:
Done. https://github.com/stacks-network/clarity-wasm/issues/439